README.md

```diff
@@ -9,16 +9,17 @@ Atlas is a permissionless and modular smart contract framework for Execution Abs
 A frontend or API wishing to participate can integrate with Atlas by completing three steps:
 
 1. Embed the Atlas SDK into their frontend or API.
-2. Create and publish a DAppControl contract.
-3. Interact with the Atlas contract to initialize the DAppControl contract and link it to the Atlas SDK on their frontend or API.
+2. Choose an Operations Relay.
+3. Create and publish a DAppControl contract.
+4. Interact with the Atlas contract to initialize the DAppControl contract and link it to the Atlas SDK on their frontend or API.
 
 ### Network Overview
 
-Atlas is infrastructure-agnostic; each app may choose how the app-designated bundler may aggregate the User Operations and Solver Operations. Examples include:
-1. **On Chain**: When gas cost is not an issue, the entire auction can take place on chain (or rollup).
-2. **On Another Chain**: So many decentralized networks - it'd be a shame to use only one.
+Atlas is infrastructure-agnostic; each app may choose how the User Operations and Solver Operations are aggregated. Examples include:
+1. **On Chain**: When gas cost is not an issue, Solver Operations may be sent on-chain, and then aggregated by any party.
```

Comment:
can we add "including a smart contract" ?

Reply:
Added it